### ISSUE TITLE
gesture swipe icons does not exist even it exists  in directory

### ISSUE THREAD

**Issue by abdennour** [react-native-training/react-native-elements#615]
<img width="974" alt="screen shot 2017-09-24 at 05 57 21" src="https://user-images.githubusercontent.com/1788384/30779051-4934c1bc-a0ed-11e7-8024-5b65c7228e2e.png">

`gesture-swipe-*` icons does not work . Seems you need a sync with dependency `react-native-vector-icons` . 

```js
<Icon 
    name="gesture-swipe-left"
     type="material-community" 
     color={'red'} />
```

**Comment by iRoachie** [comment]
Thanks for this @abdennour. We just need to update our version of react-native-vector-icons. The icons you mentioned were released in version 4.3 (https://github.com/oblador/react-native-vector-icons/commit/7396399d3294f93a449a29ccc8fbfc5e58288624), and we're on version 4.2

**Comment by iRoachie** [comment]
Fixed in v18.0.0
